src/Navigation/index.tsx

```diff
@@ -43,6 +44,11 @@ const Navigator: React.FC = () => {
     appState.current,
   );
 
+  React.useEffect(() => {
+    const ampInstance = Amplitude.getInstance();
+    ampInstance.init('b628782272a6f671076be09a7b6fb6b7');
```

Comment:
Might need to move the API key to an environmental variable like so: `AMPLITUDE_KEY={{API_KEY}}` in both the config files (`.env.dev` and `.env.prod`) and import it into the index file.

``` js
import config from 'react-native-config';

...
// other code
...

React.useEffect(() => {
    const ampInstance = Amplitude.getInstance();
    ampInstance.init(config.AMPLITUDE_KEY);
}, [])
```